### PR TITLE
setup-dev-environment.md: - Corrected the signup process for codio account

### LINKED ISSUE #113: Sign up walkthrough for Codio account is invalid
### Solution  

Sign up can be simplified by directly visiting Duckduckgo-Codio organisation signup link and then authorizing with github.

### PR DESCRIPTION
**Pull request by vigzmv**
Fixes #113 

setup-dev-environment.md: - Corrected the signup process for codio account and joining DuckDuckGo orgainisation on codio

### CODE REVIEW

File: welcome/setup-dev-environment.md [hunk 1]
```diff
@@ -56,28 +56,22 @@ Before we work on the Instant Answer codebase, we first need a development machi
 
 > Codio is the best and most popular option among contributors. If you need to develop locally, we have [alternate development environments](http://docs.duckduckhack.com/resources/other-dev-environments.html) available.
 
-### 2a. Sign Up for a Codio Account
+### 2a. Sign Up for a Account and Join the DuckDuckGo Organization on Codio
```
Comment (tagawa) [inline welcome/setup-dev-environment.md]: Please change `a Account` to `an Account`

File: welcome/setup-dev-environment.md [hunk 2]
```diff
@@ -56,28 +56,22 @@ Before we work on the Instant Answer codebase, we first need a development machi
...
 
-1. Go to https://codio.com and click "**Free Trial**", at the top right corner.
-2. Select "**Individual**" toward the bottom.
-3. Click the green "**GitHub**" icon.
-4. If you aren't already signed into GitHub, enter your GitHub login details and then click "**Sign In**".
-5. Click "**Authorize application**" to continue.
-6. In the new screen, enter the required details and click "**Create Account**".
+1. [Click this link](https://codio.com/p/signup?orgToken=Ax-OB3tU4sdNAG8axJBYcjNqR04) and you will be taken to the codio signup page.
+2. If you already have a codio account, sign in and move on to the next step.
+2. Click the green "**GitHub**" icon.
```
Comment (tagawa) [inline welcome/setup-dev-environment.md]: There are two "step 2" here - this line should be step 3, etc.

File: welcome/setup-dev-environment.md [hunk 3]
```diff
@@ -56,28 +56,22 @@ Before we work on the Instant Answer codebase, we first need a development machi
...
+3. If you aren't already signed into GitHub, enter your GitHub login details and then click "**Sign In**".
+4. Click "**Authorize application**" to continue.
+5. In the new screen, enter the required details and click "**Create Account**".
+6. You'll be added to our organization, which gives you a professional Codio setup free of charge.
+7. You should see a confirmation message at the bottom of your Codio screen after clicking.
```
Comment (tagawa) [inline welcome/setup-dev-environment.md]: I couldn't see a confirmation message (I probably just missed it) so maybe we can change step 7 to:
`7. You can check that it worked by looking for DuckDuckGo in the organization list in your [Codio settings page](https://codio.com/home/settings/).`

**Review by tagawa — CHANGES_REQUESTED** [review]

**Comment by tagawa** [comment]
Well spotted @vigzmv! I hadn't realised it had become easier to join Codio, so this is good news for all new contributors. Thank you for updating the instructions - I just made a few comments.

**Comment by vigzmv** [comment]
setup-dev-environment.md - Requested changes applied :)

**Comment by moollaza** [comment]
@vigzmv thanks for the updates! This LGTM :+1:

**Comment by csetarun** [comment]
Unable to login with the invitation code provided. It is expired.

**Pull request merged**